chains/substrate/config.go

```diff
@@ -11,10 +11,15 @@ import (
 	"github.com/centrifuge/go-substrate-rpc-client/v4/signature"
 	"github.com/creasty/defaults"
 	"github.com/mitchellh/mapstructure"
+	"github.com/sygmaprotocol/sygma-core/relayer/message"
 
 	"github.com/ChainSafe/chainbridge-core/config/chain"
 )
 
+const (
+	FungibleTransfer message.MessageType = "FungibleTransfer"
```

Comment:
Can we use this for both ERC20 and Fungible transfers and put it somewhere where it makes sense for both (outside network specific files)